sky/clouds/aws.py

```diff
@@ -31,6 +33,9 @@ def _run_output(cmd):
     return proc.stdout.decode('ascii')
 
 
+DEFAULT_AMI_GB = 45
```

Comment:
Curious - should `ami_size` be a field in the catalog (images.csv) and we can read image size from there? If you agree and think it's better done in another PR, feel free to leave a TODO here.